lxd/device/config/devices_sort.go

```diff
@@ -19,6 +19,15 @@ func (devices DevicesSortable) Less(i, j int) bool {
 
 	// First sort by types.
 	if a.Config["type"] != b.Config["type"] {
+		// Get NIC devices first (needed for VMs).
```

Comment:
@stgraber please can this comment be expanded to explain why we need the NICs to come first so that the first user device NIC gets PCI bus 5 and so gets a consistent name of enp5s0.

Reply:
Tweaked comment